File: project2/Team1/spring/src/main/java/bitc/full502/spring/domain/entity/Flight.java
package bitc.full502.spring.domain.entity;

import jakarta.persistence.*;
import lombok.*;

import java.time.LocalTime;

@Getter @Setter @NoArgsConstructor @AllArgsConstructor @Builder
@Entity @Table(name = "flight")
public class Flight {
    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(length = 20)
    private String type;

    @Column(name = "fl_no", length = 20, nullable = false)
    private String flNo;

    @Column(length = 50)
    private String airline;

    @Column(length = 10)
    private String dep;

    @Column(name = "dep_time")
    private LocalTime depTime;

    @Column(length = 10)
    private String arr;

    @Column(name = "arr_time")
    private LocalTime arrTime;

    @Column(length = 40)
    private String days;

    @Column(name = "total_seat")
    private Integer totalSeat = 20;

}

File: project2/Team1/android/app/src/main/java/bitc/fullstack502/android_studio/model/Flight.kt
package bitc.fullstack502.android_studio.model

import java.io.Serializable

data class Flight(
    val id: Long,
    val type: String?,
    val flNo: String,
    val airline: String?,
    val dep: String,
    val depTime: String,
    val arr: String,
    val arrTime: String,
    val days: String,
    val totalSeat: Int
) : Serializable   // 🔥 Serializable 추가


